appveyor.yml

```diff
@@ -55,6 +55,12 @@
       - nodejs_version: 12
         GYP_MSVS_VERSION: 2017
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      - nodejs_version: 13
```

Comment:
I don't think you need 13 anymore now that 14 is out. This will just slow down the build for no real benefit.

Reply:
yeah, basically we don't need any changes in this file. We are not using appveyor for deployments.